### DISCUSSION TITLE
Escape those attribute values!

### DISCUSSION THREAD

**Discussion by markuswustenberg** [maragudk/gomponents#81] [General]
v0.17.0 brings a security update, where the library now makes sure to escape attribute values. So feel free to pass user-generated values to `Attr` now. Special thanks to @philippta for reporting and guidance. 😎

Also, there are documentation updates, so there should be runnable examples on godoc soon.

Also, two new helper components made it into the `components` package: `LinkStylesheet` and `LinkPreload`. Enjoy!

<hr /><em>This discussion was created from the release <a href='https://github.com/maragudk/gomponents/releases/tag/v0.17.0'>Escape those attribute values!</a>.</em>

**Comment by markuswustenberg** [comment]
Thank you again @philippta, your fix is now released. 😊

**Reply by philippta** [reply]
Awesome work man! 🥳
